src/webview.cpp

```diff
@@ -75,6 +79,13 @@ bool WebView::eventFilter(QObject *src, QEvent *e)
         if (we->modifiers() == Qt::ControlModifier)
             return true;
     }
+    if (e->type() == QEvent::MouseButtonRelease) {
+        auto me = static_cast<QMouseEvent *>(e);
+        if (me->modifiers() == Qt::ControlModifier && !m_linkHovered.startsWith("zim://")) {
```

Comment:
There is two way to open a link in a new tab : with Ctrl-click and middle click.
You are catching the ControlModifier but you must also catch the middleButton.